src/Compiler/Microsoft.AspNetCore.Razor.Language/test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/AtTransitions/TestComponent.codegen.cs

```diff
@@ -27,10 +27,9 @@ protected override void BuildRenderTree(global::Microsoft.AspNetCore.Components.
 #line hidden
 #nullable disable
 
-            __builder.AddContent(0, 
 #nullable restore
-#line (3,6)-(3,7) "x:\dir\subdir\Test\TestComponent.cshtml"
-x
+#line (3,6)-(3,7) 24 "x:\dir\subdir\Test\TestComponent.cshtml"
+__builder.AddContent(0, x
```

Comment:
Can we add some tests that verify the actual sequence points emitted?

Reply:
Alright, consider me suitably nerd sniped 😆 I added the ability to diff sequence points and added a couple of diffs to existing tests.